File: circulo.java
public class circulo {
    private double radio;
    
    public circulo(double radio){
        this.radio = radio;
    }

    public double CalcularArea() {
        return Math.PI * radio * radio;
    }

    public double CalcularPerimetro (){
        return 2 * Math.PI * radio;
    }
        
    public void CambiarRadio( double NuevoRadio){
        this.radio = NuevoRadio;
    }

    public static void main(String[] args){

        circulo miCirculo = new circulo(5.0); 

        double area = miCirculo.CalcularArea();
        System.out.println("El area del circulo es: " + area);

        double perimetro = miCirculo.CalcularPerimetro();
        System.out.println("El perimetro del circulo es: " + perimetro);
    }
}

File: circulo.cpp
#include <iostream>
#include <cmath>

class Circulo {
private:
    double radio;

public:
    // Constructor
    Circulo(double r) : radio(r) {}

    // Métodos
    double calcularArea() {
        return M_PI * radio * radio;
    }

    double calcularPerimetro() {
        return 2 * M_PI * radio;
    }

    void cambiarRadio(double nuevoRadio) {
        radio = nuevoRadio;
    }
};

int main() {

    Circulo miCirculo(5.0); 

    double area = miCirculo.calcularArea();
    std::cout << "El área del círculo es: " << area << std::endl;

    double perimetro = miCirculo.calcularPerimetro();
    std::cout << "El perímetro del círculo es: " << perimetro << std::endl;

    return 0;
}
